diktat-rules/src/main/resources/diktat-analysis-huawei.yml

```diff
@@ -138,7 +138,7 @@
   enabled: true
   configuration:
     isCopyrightMandatory: true
-    copyrightText: ' Copyright (c) Huawei Technologies Co., Ltd. 2012-2021. All rights reserved.'
+    copyrightText: ' Copyright (c) Huawei Technologies Co., Ltd. 2012-;@currYear;. All rights reserved.'
```

Comment:
and what's about other inspections? Do we need to add any?

Reply:
For reinsuring I did similar updates in all configs, where do we have `copyrightText`